webextensions/api/menus.json

```
@@ -72,6 +66,27 @@
               }
             }
           },
+          "action": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "91"
```

```suggestion
                  "version_added": "109"
```

While the work landed in 91, the API is tied to MV3, which was only enabled by default in 109.

The action API itself has the BCD set to 101, but that was because we announced the "dev preview" back then. In retrospect, we should have used `"version_added": "preview"` for these instead of a fixed version.